modules/ROOT/pages/mongodb/mongodb-connector-6-0-upgrade-migrate.adoc

```diff
@@ -0,0 +1,344 @@
+| N/A
+| Get file content
+a|* File id (Object)
+| N/A
+| No changes
+a|* File id (InputStream)
+| N/A
+| Remove files
+a|* File id (Object)
+| N/A
+| No changes
+a|* File id (InputStream)
+| N/A
+|===
+
+== Changed Operations Metadata
+
+MongoDB 5.x generated operations metadata when the user provided a set of documents per collection from which to take the attributes. MongoDB 6.0 generates metadata automatically, based on the latest document in each collection.
+
+=== Metadata in MongoDB v6.0
+
+[%header,cols="34%,33%,33%"]
+|===
+|Operation Name	| Input Metadata	|Output Metadata
+
+| Insert Document
+|Document
+| Resolved dynamically based on the selected value of the collection parameter. The connector adds the latest document of the given collection and uses the document's structures structure as input/output metadata.
+
+| Insert Documents
+|Document
+|Resolved dynamically based on the selected value of the collection parameter. The connector adds the latest document of the given collection and uses the document's structures structure as input/output metadata.
+
+| Update Documents
+| N/A
+| JSON that contains the following structure: { "matched" : int, "modified" : int, upsertedId: JSON }
+
+| Remove Documents
+| Query: JSON Object
+| N/A
+
+| Count Documents
+| Query: JSON Object
+| N/A
+
+| Find Documents
+| Query: JSON Object
+| Resolved dynamically based on the selected value of the collection parameter. The connector uses the structure of the first document in the given collection.
+*Reviwers, in the last sentence, what does the connector use the structure of the first document for?*
+
+| Create File
+| N/A
+a|** A GridFS JSON object with the following structure:
+* JSON Id
+* String fileName
+* Long Length
+* Int ChunkSize
+* Datetime uploadDate
+* JSON Metadata
+
+| Find Files
+| N/A
+a|** List<InputStream> +
+Each InputStream is a GridFS JSON object with the following attributes:
+* JSON Id
+* String fileName
+* Long Length
+* Int ChunkSize
+* Datetime uploadDate
+* JSON Metadata
+|===
+
+== Removed Operations
+
+The following operations were removed from the MongoDB connector:
+
+[%header,cols="15%,35%,15%,35%"]
+|===
+2+|Removed from MongoDB 5.x 2+| Can be reproduced in MongoDB 6.0.0 through
+|Name	| Description	|Name	|Description
+|Incremental dump | Executes an incremental dump of the database. | Dump | Use the Dump operation.
+|Restore | Takes the output from the dump and restores it | Restore from file or Restore from directory | These operations take the output from the dump file or directory and restore it.
+|Update documents by function | Update documents using a Mongo function | N/A | N/A
+|Update documents by functions | Update documents using one or more Mongo functions | N/A | N/A
+|Find one and update document | Finds and updates the first document that matches a given query | N/A | This operation's functionality can be reproduced by combining other operations
+*Reviewers, can you provide examples*
```

Comment:
First, the Find Documents operation must be invoked with a limit of 1 document, then the Update Document function must be invoked specifying the ID of the document obtained previously.

Reply:
Is this one example or the only way to reproduce the operation's functionality?

Reply:
This is an example but there could be another combination of operations that can do this.